tabbycat/privateurls/utils.py

```diff
@@ -50,16 +51,15 @@ def send_randomised_url_emails(request, tournament, queryset, url_name, url_key_
     for instance in queryset:
         url_key = url_key_function(instance)
         email = instance.email
-
         path = reverse_tournament(url_name, tournament, kwargs={'url_key': url_key})
         url = request.build_absolute_uri(path)
 
-        formatted_subject = subject
-        formatted_message = message.replace('<NAME>', instance.name).replace('<URL>', url)
+        variables = {'NAME': instance.name, 'URL': url}
         if hasattr(instance, 'team'):
-            formatted_message = formatted_message.replace('<TEAM>', instance.team.short_name)
+            variables['TEAM'] = instance.team.short_name
 
-        messages.append((formatted_subject, formatted_message, settings.DEFAULT_FROM_EMAIL, [email]))
+        formatted_message = message.render(Context(variables))
+        messages.append((subject, formatted_message, settings.DEFAULT_FROM_EMAIL, [email]))
```

Comment:
Did we forget to render the subject template here? The emails I'm getting when testing this have subject line: "<django.template.base.Template object at 0x7fa7c87ac710>". (The other emails look fine.)